tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -33,6 +33,12 @@ arp/test_arp_dualtor.py::test_proxy_arp_for_standby_neighbor:
     conditions:
       - "release not in ['202012']"
 
+arp/test_unknown_mac.py:
+  skip:
+    reason: "Not supported on cisco-8000 all releases asics"
```

Comment:
instead of 'release asics', can reason be changed to 'Not supported on cisco-8000 all releases & asics'  or just ''Not supported on cisco-8000 platform'. Rest looks good

Reply:
Changed the reason